packages/fast-components-styles-msft/src/slider/index.ts

```diff
@@ -1,264 +1,260 @@
-
-        slider_thumb__upperValue: {},
-
-        slider_track: {},
-
-        slider_backgroundTrack: {
-            borderRadius: "2px",
-            background: neutralOutlineRest,
+    },
+    slider__horizontal: {
+        "&$slider": {
+            width: "100%",
+            minHeight: thumbSize,
+            minWidth: minSize,
         },
-
-        slider_foregroundTrack: {
-            transition: "all .04s linear",
-            borderRadius: "2px",
-            background: neutralForegroundHint,
+        "& $slider_layoutRegion": {
+            margin: format(`0 {0}`, halfThumbSize),
+            gridTemplateRows: format(`{0} 1fr`, thumbSize),
         },
-
-        slider__disabled: {
-            opacity: ".3",
+        "& $slider_thumb": {
+            alignSelf: "start",
         },
-
-        slider__horizontal: {
-            "&$slider": {
-                minHeight: "36px",
-                minWidth: "128px",
+        "& $slider_thumb__upperValue": {
+            transform: format(`translateX({0})`, halfThumbSize),
+        },
+        "& $slider_thumb__lowerValue": {
+            transform: format(`translateX(-{0})`, halfThumbSize),
+        },
+        "& $slider_track": {
+            alignSelf: "start",
+            height: thumbSize,
+            width: "100%",
+        },
+        "& $slider_backgroundTrack": {
+            marginTop: trackOffset,
+            alignSelf: "start",
+            height: toPx(designUnit),
+            left: trackOverhang,
+            right: trackOverhang,
+        },
+        "& $slider_foregroundTrack": {
+            marginTop: trackOffset,
+            alignSelf: "start",
+            height: toPx(designUnit),
+        },
+        "&$slider__modeAdjustLower": {
+            "& $slider_foregroundTrack": {
+                marginRight: trackOverhang,
             },
-
-            "& $slider_layoutRegion": {
-                margin: "0 10px 0 10px",
-                gridTemplateRows: "20px 1fr",
+        },
+        "&$slider__modeAdjustUpper": {
+            "& $slider_foregroundTrack": {
+                marginLeft: trackOverhang,
             },
-
-            "& $slider_thumb": {
-                alignSelf: "start",
+        },
+    },
+    slider__vertical: {
+        "&$slider": {
+            height: "100%",
+            minHeight: minSize,
+            minWidth: thumbSize,
+        },
+        "& $slider_thumb": {
+            justifySelf: "start",
+        },
+        "& $slider_layoutRegion": {
+            margin: format(`{0} 0`, halfThumbSize),
+            gridTemplateColumns: format(`{0} 1fr`, thumbSize),
+        },
+        "& $slider_thumb__upperValue": {
+            transform: format(`translateY(-{0})`, halfThumbSize),
+        },
+        "& $slider_thumb__lowerValue": {
+            transform: format(`translateY({0})`, halfThumbSize),
+        },
+        "& $slider_track": {
+            justifySelf: "start",
+            marginLeft: trackOffset,
```

Comment:
ltr? same with all these other marginLefts

Reply:
This is not covered in the rtl section below because it comes for free with the way the css grid is flipped.